sensor/common/centralclient/client.go

```diff
@@ -99,21 +103,21 @@ func NewClient(endpoint string) (*Client, error) {
 	}, nil
 }
 
-// GetMetadata returns Central's metadata
-func (c *Client) GetMetadata(ctx context.Context) (*v1.Metadata, error) {
-	resp, _, err := c.doHTTPRequest(ctx, http.MethodGet, metadataRoute, nil, nil)
+// GetPing pings Central.
+func (c *Client) GetPing(ctx context.Context) (*v1.PongMessage, error) {
+	resp, _, err := c.doHTTPRequest(ctx, http.MethodGet, pingRoute, nil, nil)
 	if err != nil {
-		return nil, errors.Wrap(err, "receiving Central metadata")
+		return nil, errors.Wrap(err, "pinging Central")
 	}
 	defer utils.IgnoreError(resp.Body.Close)
 
-	var metadata v1.Metadata
-	err = jsonutil.JSONReaderToProto(resp.Body, &metadata)
+	var pong v1.PongMessage
+	err = jsonutil.JSONReaderToProto(resp.Body, &pong)
 	if err != nil {
-		return nil, errors.Wrapf(err, "parsing Central %s response with status code %d", metadataRoute, resp.StatusCode)
+		return nil, errors.Wrapf(err, "parsing Central %s response with status code %d", pingRoute, resp.StatusCode)
```

Comment:
super nit to avoid shadowing:
```go
if err := = jsonutil.JSONReaderToProto(resp.Body, &pong); err != nil {
		return nil, errors.Wrapf(err, "parsing Central %s response with status code %d", pingRoute, resp.StatusCode)
```